ivy/functional/frontends/torch/func_wrapper.py

```diff
@@ -130,3 +130,55 @@ def outputs_to_native_arrays_torch(*args, **kwargs):
         return ret
 
     return outputs_to_native_arrays_torch
+
+
+def torch_to_numpy_style_args(func):
+    """Convert argument names from PyTorch style to NumPy style."""
+    numpy_compatibility_arg_names = {
+        "dim": ["axis"],
+        "keepdim": ["keepdims"],
+        "input": ["x", "a", "x1"],
+        "other": ["x2"],
+    }
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        # Translate PyTorch-style args to NumPy-style args
+        new_kwargs = {}
+        for k, v in kwargs.items():
+            ivy_names = numpy_compatibility_arg_names.get(k, [k])
+            for ivy_name in ivy_names:
+                new_kwargs[ivy_name] = v
+        # Translate argument names explicitly
+        for pytorch_name, numpy_names in numpy_compatibility_arg_names.items():
+            for numpy_name in numpy_names:
+                if numpy_name in new_kwargs:
+                    new_kwargs[pytorch_name] = new_kwargs.pop(numpy_name)
+        return func(*args, **new_kwargs)
+
+    return wrapper
+
+
+# numpy_compatibility_arg_names = {
+#     "dim": ["axis"],
+#     # "keepdim": ["keepdims"],
+#     # "input": ["x", "a", "x1"],
+#     # "other": ["x2"],
+# }
+
+# def torch_to_numpy_style_args(func):
+#     """
+#     Decorator for translating argument names from PyTorch style to NumPy style.
+#     """
+#     @functools.wraps(func)
+#     def wrapper(*args, **kwargs):
+#         # Translate PyTorch-style args to NumPy-style args
+#         new_kwargs = {}
+#         for k, v in kwargs.items():
+#             ivy_names = numpy_compatibility_arg_names.get(k, [k])
+#             for ivy_name in ivy_names:
+#                 new_kwargs[ivy_name] = v
+#         if 'axis' in new_kwargs:
+#             new_kwargs['dim'] = new_kwargs.pop('axis')
+#         return func(*args, **new_kwargs)
+#     return wrapper
```

Comment:
Also, I guess you forgot to remove the commented out part?

Reply:
Sorry! It was part of an earlier testing code !